homeassistant/helpers/selector.py

```diff
@@ -120,4 +137,28 @@ class TargetSelector(Selector):
     Value should follow cv.ENTITY_SERVICE_FIELDS format.
     """
 
-    CONFIG_SCHEMA = vol.Schema({"entity": {"domain": str, "device_class": str}})
+    CONFIG_SCHEMA = vol.Schema(
+        {
+            vol.Optional("entity"): vol.Schema(
+                {
+                    vol.Optional("domain"): str,
+                    vol.Optional("device_class"): str,
+                    vol.Optional("integration"): str,
+                }
+            ),
+            vol.Optional("device"): vol.Schema(
+                {
+                    vol.Optional("integration"): str,
+                    vol.Optional("manufacturer"): str,
+                    vol.Optional("model"): str,
+                }
+            ),
+        }
+    )
+
+
+@SELECTORS.register("sequence")
+class SequenceSelector(Selector):
+    """Selector of a sequence."""
```

Comment:
Sequence, is that for scripts?

Reply:
We should improve this description, it's not clear.

Reply:
I actually think `Sequence` is not a good word. We refer to it as scripts internally. I think just `script` makes sense.

Reply:
I doubted between action/script and sequence...

But a script consist of a sequence (the part this is about) and metadata (run mode, name). In an automation we call the sequence an action... so... Idk?

Reply:
ScriptSequence then? Because `Sequence` without context is too generic.